src/fal/run_scripts.py

```diff
@@ -96,6 +108,42 @@ def ref_resolver(target_model_name: str, target_package_name: Optional[str] = No
     return ref_resolver
 
 
+def _get_firestore_writer(model: ParsedModelNode, manifest: Manifest):
+    # Use the application default credentials
```

Comment:
Did you test with the different ways of authenticating shown in dbt's documentation? In case you didn't, add a TODO at least.